tests/integration/scripts/test_validate_integration.py

```diff
@@ -94,3 +94,64 @@ async def test_validate_against_source_gets_old_tvars(
         flow_file.read_text().replace('P1Y = foo', 'P1Y = {{FOO}}'))
     with pytest.raises(Jinja2Error):
         validate(src_dir)
+
+
+@pytest.mark.compatmode
```

Comment:
I don't think this actually tests Cylc 7 compatibility mode?

Reply:
It is!
```
[scheduling]
  [[dependencies]]
    graph = foo
```